lldb/source/Plugins/SymbolFile/DWARF/DWARFDIE.cpp

```diff
@@ -376,7 +378,8 @@ lldb_private::Type *DWARFDIE::ResolveTypeUID(const DWARFDIE &die) const {
   return nullptr;
 }
 
-static CompilerContext GetContextEntry(DWARFDIE die) {
+static CompilerContext GetContextEntry(DWARFDIE die,
+                                       bool complete_template_names) {
```

Comment:
`complete_template_names` is a good name for this parameter but there's quite a few overloaded terms for "completing" types/etc. in this part of the code already. Could we use something else instead? E.g., `derive_template_names`, or something like that.

Also this is only set to `true` when called with "simple template names". So on first glance of the API a user might expect `simple template names => !complete_template_names`

If we can't find anything more suitable i'm also fine with this

Reply:
derive_template_names SGTM, and it also sort of gives a hint that the operation will be more expensive, which seems nice.